soccer/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ROBOCUP_LIB_SRC
     "motion/TrapezoidalMotion.cpp"
     "NewRefereeModule.cpp"
     "planning/CompositePath.cpp"
-    "planning/DirectTargetPathPlanner.cpp"
+    #"planning/DirectTargetPathPlanner.cpp"
```

Comment:
Ok cool.  Yeah I think we should just leave MotionControl stuff alone at the moment, then we can get to it later once we have the more path-oriented stuff in place.